README.rst

```diff
@@ -17,5 +17,47 @@ Database properties under ``pki`` key inside ``configuration`` database:
 - ``LetsEncryptRenewDays``: days to the expiration, the certificate will be renewd when the condition is met
 - ``LetsEncryptMail``: (optional) registration mail for LE notifications
 - ``LetsEncryptDomains``: comma-separated list of domains added to certificate SAN field
+- ``LetsEncryptChallenge``: challenge to use for validating the certificate, default is ``http``.
+  It accepts also values like ``dns-<provider>``. Where ``<provider>`` is the name of the DNS provider.
+  See full list of available DNS plugins by executing ``certbot -h certonly``.
+  More info at https://certbot.eff.org/docs/using.html?highlight=dns#dns-plugins.
 
+DNS challenge
+=============
 
+To use the DNS challenge, follow these steps:
+
+- install the required certbot plugin plugin using yum; to see the list of available package use ``yum search certbot-dns``
+- set ``LetsEncryptChallenge`` property to correct DNS plugin
+- configure all required properties accordingly to plugin documentation
+
+When using the dns challenge, make sure to set extra properties accordingly to certbot configuration.
+All properties for the dns challenge should be in the form ``LetsEncrypt_<certbot_option>``, where
+``<certbot_option>`` is the option specific to certbot DNS plugin.
+
+Digitial Ocean example
+----------------------
+
+1. Install the plugin:
+
+   ::
+
+     yum install python2-certbot-dns-digitalocean
+
+2. Configure the challenge type:
+
+   ::
+
+     config setprop pki LetsEncryptChallenge dns-digitalocean
```

Comment:
```suggestion
     config setprop pki LetsEncryptChallenge dns_digitalocean
```

Below the underscore is used: which one is correct?

Reply:
The one with the normal dash